src/seedu/addressbook/AddressBook.java

```diff
@@ -141,19 +157,19 @@
      * used by the internal String[] storage format.
      * For example, a person's name is stored as the 0th element in the array.
      */
-    private static final int PERSON_DATA_INDEX_NAME = 0;
-    private static final int PERSON_DATA_INDEX_PHONE = 1;
-    private static final int PERSON_DATA_INDEX_EMAIL = 2;
+	// private static final int PERSON_DATA_INDEX_NAME = 0;
+	// private static final int PERSON_DATA_INDEX_PHONE = 1;
+	// private static final int PERSON_DATA_INDEX_EMAIL = 2;
 
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.